yamlExamples/AgentExamples.md

```diff
@@ -0,0 +1,73 @@
+### Docker image example 1
+
+```yaml
+agent:
+  label: my_label
+  customWorkspace: path_to_workspace
+  dockerImage: maven:3-alpine
+  args: -v /tmp:/tmp
```

Comment:
Need example with multiple args

Reply:
I cannot find an example of the declarative pipeline which has multiple arguments for docker image to relate with.